modules/openapi-generator/src/main/resources/typescript-nestjs/configuration.mustache

```diff
@@ -13,7 +13,7 @@ export class Configuration {
     apiKeys?: {[ key: string ]: string};
     username?: string;
     password?: string;
-    accessToken?: string | (() => string);
+    accessToken?: string | Promise<string> | (() => string | Promise<string>);
```

Comment:
the type in `ConfigurationParameters` should be adjusted as well, right?

Reply:
yeah I just noticed as well